Cargo.toml

```diff
@@ -19,10 +19,10 @@ tokio-stream = "0.1.12"
 tokio-tungstenite = "0.14"
 tracing = "0.1.23"
 tracing-subscriber = "0.2.15"
+testcontainers = "0.14.0"
```

Comment:
why is this not a dev dependency anymore? ideally it'll only be used for testing

Reply:
because mod `test_utils` is now exported as a module in the `crate` (lib.rs), and being used as an api in the examples. 

The reason it worked before when in macro form, is because in the macro calls it's as if the code is physically copied into where the macro is called. That's why you needed to have the imports of the `use testcontainers::{clients, core::WaitFor, images::generic::GenericImage};` anywhere it is called. This is not very convenient as you are referencing more and more types in the macro code.